arkane/kinetics.py

```diff
@@ -284,8 +284,8 @@ def save(self, outputFile):
             f.write('# krev (TST+T) = {0} \n\n'.format(kineticsrev))
 
         # Reaction path degeneracy is INCLUDED in the kinetics itself!
-        string = 'kinetics(label={0!r}, kinetics={1!r})'.format(reaction.label, reaction.kinetics)
-        f.write('{0}\n\n'.format(prettify(string)))
+        _string = 'kinetics(label={0!r}, kinetics={1!r})'.format(reaction.label, reaction.kinetics)
```

Comment:
I would probably name it something more informative, like `rxn_str` or `kinetics_str`.